From 8edcca5773f62d836e0456ea2575008c5f0384e4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Sat, 30 Jul 2011 22:30:39 +0800
Subject: [PATCH] Revert "kernel: printk: Add non exported function for
 clearing the log ring buffer"

This reverts commit 90bb75ece868932904865ea46e61b78e5a987224.
---
 kernel/printk.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/kernel/printk.c b/kernel/printk.c
index 23042b1b3a08..16fbcf7a2de3 100644
--- a/kernel/printk.c
+++ b/kernel/printk.c
@@ -268,14 +268,6 @@ static int log_buf_get_len(void)
 	return logged_chars;
 }
 
-/*
- * Clears the ring-buffer
- */
-void log_buf_clear(void)
-{
-	logged_chars = 0;
-}
-
 /*
  * Copy a range of characters from the log buffer.
  */
-- 
2.35.3

